documentation/docs/getting_started/python.mdx

```diff
@@ -114,3 +114,212 @@ print(account)
+
+:::info
+
+In this guide we will rename the `iota-wallet` to `iota-wallet-production`, but you can use whatever you want.
+
+:::
+
+In `bindings/python/native/Cargo.toml` change both names:
+
+ ```diff
+  [package]
+- name = "iota-wallet-python"
++ name = "iota-wallet-production"
+  version = "0.2.0"
+  authors = ["IOTA Stiftung"]
+  edition = "2021"
+  description = "Python bindings for the IOTA wallet library"
+  documentation = "https://wiki.iota.org/wallet.rs/welcome"
+  homepage = "https://www.iota.org/"
+  repository = "https://github.com/iotaledger/wallet.rs"
+  license = "Apache-2.0"
+  keywords = ["iota", "wallet", "transaction", "python"]
+  categories = ["cryptography::cryptocurrencies"]
+
+  [lib]
+- name = "iota_wallet"
++ name = "iota_wallet_production"
+  crate-type = ["cdylib"]
+
+  [dependencies]
+  chrono = { version = "0.4.19", default-features = false }
+  dict_derive = "0.3.0"
+  futures = { version = "0.3.17", default-features = false }
+  hex = { version = "0.4.3", default-features = false }
+  iota-client = { git = "https://github.com/iotaledger/iota.rs", rev = "47e5f6684e2c59237c5fac88be85950a6e5eec4f", default-features = false, features = ["async", "mqtt", "tls"] }
+  iota-wallet = { path = "../../../", version = "0.2.0", features = ["stronghold", "ledger-nano", "ledger-nano-simulator"] }
+  once_cell = { version = "1.8.0", default-features = false }
+  serde = { version = "1.0.130", default-features = false, features = ["derive"] }
+  serde_json = { version = "1.0.68", default-features = false }
+  tokio = { version = "1.12.0", default-features = false }
+  url = { version = "2.2.2", default-features = false }
+
+  [dependencies.pyo3]
+  version = "0.13"
+  features = ["extension-module", "abi3-py36"]
+ ```
+
+In `bindings/python/native/setup.py`:
+```diff
+setup(
+-   name="iota_wallet",
++   name="iota_wallet_production",
+    version="0.2.0",
+    classifiers=[
+        "License :: SPDX-License-Identifier ::  Apache-2.0",
+        "Development Status :: 0.1.0 - Alpha",
+        "Intended Audience :: Developers",
+        "Programming Language :: Python",
+        "Programming Language :: Rust",
+        "Operating System :: POSIX",
+        "Operating System :: MacOS :: MacOS X",
+    ],
+    packages=["iota_wallet"],
+    rust_extensions=[
+        RustExtension(
+            "iota_wallet.iota_wallet",
+            rustc_flags=get_py_version_cfgs(),
+            debug=False,
+        ),
+    ],
+    include_package_data=True,
+    zip_safe=False,
+)
+```
+
+And in `bindings/python/native/src/lib.rs`:
+```diff
+/// IOTA Wallet implemented in Rust and binded by Python.
+#[pymodule]
+- fn iota_wallet(_py: Python, m: &PyModule) -> PyResult<()> {
++ fn iota_wallet_production(_py: Python, m: &PyModule) -> PyResult<()> {
+      m.add_class::<AccountInitialiser>()?;
+      m.add_class::<AccountHandle>()?;
+      m.add_class::<SyncedAccount>()?;
```

Comment:
True, here it seems a bit extreme. I did it for iota.rs but than thought it would just be nicer to have some context but than just included the whole section as it is not much.

Reply:
Shortened the diffs